web/packages/core/src/ruffle-player.ts

```diff
@@ -1900,6 +1900,7 @@ export class RufflePlayer extends HTMLElement {
                 panicBody.classList.add("details");
 
                 const panicText = document.createElement("textarea");
+                panicText.setAttribute("readonly", "1");
```

Comment:
nit: I think the `.readOnly` property can be set directly, without `.setAttribute()`.